telebot/settings.py

```diff
@@ -2,3 +2,5 @@
 JOB_PLUGINS = ['remind', 'cureport']
 # List of plugins that use ConverstationHandler
 CONV_PLUGINS = ['']
+# Plugins for notification
+NOTIFICATION_PLUGINS = ['notification_to_user']
```

Comment:
For now, it should be changed to "NOTIFICATION_PLUGINS = ['notify']

Reply:
NOTIFY_PLUGINS = ['notify']